internal/ziputils/ziputils.go

```
@@ -27,19 +27,36 @@ func IsZipFile(filename string) bool {
 	base := path.Base(name)
 	return strings.HasPrefix(base, "._") || base == ".DS_Store"
 }
 
+// IsBundleZip reports whether the archive has a root entry named "manifest.json"
+// (exact, case-sensitive). Presence only — content is checked later.
+func IsBundleZip(zipPath string) bool {
+	reader, err := zip.OpenReader(zipPath)
+	if err != nil {
+		return false
+	}
+	defer reader.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `reader.Close` is not checked

(errcheck)